media-gfx/darktable/darktable-2.4.0.ebuild

```diff
@@ -79,6 +75,9 @@ pkg_pretend() {
 }
 
 src_prepare() {
+	default
+	eapply "${FILESDIR}"/"${P}"-find-opencl-header.patch
```

Comment:
It's better to not use variables in the patch name. Often, a patch will continue to apply for several following versions.

Reply:
Contrary, I have often enough been asked why not use `${P}`, so this is a very common practice and comes down to dev's personal preference. We don't do blind version bumps anyway, but if someone still does it then the possible consequence just shifts from 'ebuild failed w/ missing patch' to 'ebuild failed to apply patch'. Does `9999` need the same modification?

In an ideal world of course you will submit this patch upstream and it will land in time before the next upstream release. Have you done that, @V3n3RiX?

Reply:
Contrary, I have often enough been asked why not use `${P}`, so this is a very common practice and comes down to dev's personal preference. We don't do blind version bumps anyway, but if someone still does it then the possible consequence just shifts from 'ebuild failed w/ missing patch' to 'ebuild failed to apply patch'. Does `9999` need the same modification?

In an ideal world of course you will submit this patch upstream and it will land in time before the next upstream release. Have you done that, @V3n3RiX?